src/main/scala/dxWDL/Main.scala

```diff
@@ -333,13 +337,13 @@ object Main extends App {
         )
     }
 
-  private def parseStreamAllFiles(s: String): Boolean = {
+  private def parseStreamFiles(s: String): StreamFiles.StreamFiles = {
     s.toLowerCase match {
-      case "true"  => true
-      case "false" => false
+      case "all"   => StreamFiles.All
+      case "false" => StreamFiles.None
       case other =>
         throw new Exception(
-            s"""|the streamAllFiles flag must be a boolean (true,false).
+            s"""|the streamFiles flag must be a boolean (true,false).
```

Comment:
the streamFiles flag must be a string ("all", "none").

Reply:
good catch